packages/next-tweet/src/api/types/media.ts

```diff
@@ -52,7 +52,7 @@ export interface MediaPhoto extends MediaBase {
 }
 
 export interface MediaAnimatedGif extends MediaBase {
-  type: 'animated_gif'
+  type: 'animated_gif' | 'video'
```

Comment:
I think this can be better named, `type: 'video'` shouldn't be inside the `MediaAnimatedGif` type.